__tests__/components/resetPassword/resetPasswordform.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import ResetPaswordForm from '../../../src/components/resetPassword/ResetPasswordForm';
+
+describe('Test reset password', () => {
+  it('form test', async () => {
+    const displayMessage = '';
+    const checkOnsubmit = jest.fn();
+    const callApiThunk = jest.fn();
+    const location = jest.fn();
+    const checkOnchange = jest.fn();
```

Comment:
Please avoid declaring a lot of variables. You can use objects.

Reply:
I let me do it now

Reply:
The errors were not mine but we will refactor the after

Reply:
cool.